### ISSUE TITLE
Keyboard tabbing on Preview resource modal

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#4019]
When keyboard tabbing through the Preview resource modal, the tab skips the expand/contract icons so the user is unable to expand and contract sections by keyboard tabbing.
